otter/convergence/service.py

```diff
@@ -440,6 +456,24 @@ def buckets_acquired(self, my_buckets):
         # the return value is ignored, but we return this for testing
         return result
 
+    def divergent_changed(self, children):
+        """
+        ZooKeeper children-watch callback that lets this service know when the
+        divergent groups have changed. If any of the divergent flags are for
+        tenants associated with this service's buckets, a convergence will be
+        triggered.
+        """
+        # It'd sure be nice if we could know *exactly which* flag was just
```

Comment:
Random non-blocking thought - can we store all the previous divergent groups, and compare it to the new divergent groups, to see which groups have been added?  Much like the current set comparison.